supplementary_style_guide/glossary_terms_conventions/general_conventions/t.adoc

```diff
@@ -1,7 +1,7 @@
 // AMQ: Added "In Red Hat AMQ, a target is"
 [[target]]
 ==== image:images/yes.png[yes] target (noun)
-*Description*: In Red Hat AMQ, a _target_ is a message's destination. This is usually a queue or topic.
+*Description*: 1) Generically, synonymous with destination, a _target_ is a file, device or any type of location to which data is moved or copied. 2) In Red Hat AMQ, a _target_ is a message's destination. This is usually a queue or topic.
```

Comment:
```suggestion
*Description*: 1) Generically, a _target_ is synonymous with destination. A _target_ can be a file, a device or any type of location to which data is moved or copied. 2) In Red Hat AMQ, a _target_ is a message's destination. The _target_ is usually a queue or topic.
```
I would split this entry into a bulleted list, as I suggested for previous similar entries.